web/wp-content/themes/bumblebee/single-listicle.php

```diff
@@ -179,7 +179,7 @@ function () {
 			<?php endif; ?>
 			<?php $section_num++; ?>
 		<?php endfor; ?>
-		<section class="content pure-g">
+		<section class="content pure-g" style="width: 100%;">
```

Comment:
@fariasf  just to know why this inline styling and does it relate to this ticket?

Reply:
When Taboola is blocked the last card messes up. I discovered the issue while working on blocking Taboola, so I fixed it right here to avoid introducing visual regressions.